.github/actions/ci/action.yml

```diff
@@ -5,19 +5,12 @@ inputs:
     description: 'Version of Lua to use for building and testing.'
     required: false
     default: '5.3'
```

Comment:
Refactored these out into an action, since it's also needed for the manual publish workflow.